src/views/LoginView.vue

```diff
@@ -1,114 +1,196 @@
 <template>
-  <v-container id="login-container" class="container">
-    <img id="logo" alt="Cuttle logo" src="/img/logo.png" />
-
-    <v-row>
-      <!-- Left side form -->
-      <v-col id="username-login-form" :cols="$vuetify.display.mdAndUp ? 6 : 12">
-        <h1 class="gradient-text">
-          {{ buttonText }}
-        </h1>
-
-        <form @submit.prevent="submitLogin">
-          <v-text-field
-            v-model="username"
-            variant="outlined"
-            :dense="$vuetify.display.mdAndDown ? true : false"
-            hint="Username"
-            data-cy="username"
-            autocomplete="username"
-            placeholder="Username"
-          />
-          <v-text-field
-            v-model="pw"
-            variant="outlined"
-            hint="Password"
-            :dense="$vuetify.display.mdAndDown ? true : false"
-            type="password"
-            data-cy="password"
-            :autocomplete="isLoggingIn ? 'current-password' : 'new-password'"
-            placeholder="Password"
-          />
-          <div id="login-button-container">
+  <div class="loginViewContainer">
+    <!--////////////////////////// 
+     /////Welcome Section ///// 
+    //////////////////////////-->
+    <section v-if="!isLoggingIn">
+      <v-container fluid id="welcome-container" class="welcomeContainer">
+        <nav class="d-flex justify-space-between align-center mb-2">
+          <img class="cardLogo" src="/img/loginView/logo-cards-behind.svg">
+          <v-btn  
+            variant="text"
+            class="text-h6"
+            @click ="$refs.usernameInput.focus()"
+          >
+            Sign up 
+            <img class="ml-2" src="/img/loginView/icon-account.svg" />
+          </v-btn>        
+        </nav>
+
+        <div class="d-flex h-75 flex-column justify-space-around align-center mt-2">
+          <h1>Welcome to Cuttle.Cards</h1>
+          <p class="text-h4 mb-4">The card game that will have you hooked from the first hand!</p>
+        
+          <div class="video-container__wrapper">
+            <div class="video-container">
+              <iframe
+                class="video-container__video"
+                src="https://www.youtube.com/embed/qOqkNbhMdsI"
+                title="Cuttle Game Tutorial -- Youtube Player"
+                allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+                allowfullscreen
+              />
+            </div>
+            <!-- Grouped for the ease of styling -->
             <v-btn
-              :loading="loading"
-              color="primary"
-              block
-              type="submit"
-              data-cy="submit"
-            >
-              {{ buttonText }}
-            </v-btn>
+              class="w-100 mt-8"
+              size='x-large'
+              to="/rules"
+              color="newPrimary"
+              data-cy="rules-link"
+            >Read The Rules</v-btn>
```

Comment:
cleaner to format the content on a newline
```suggestion
              class="w-100 mt-8"
              size='x-large'
              to="/rules"
              color="newPrimary"
              data-cy="rules-link"
            >
              Read The Rules
            </v-btn>
```